tests/unit/lbrynet_daemon/test_Downloader.py

```diff
@@ -103,6 +106,17 @@ def test_init_exception(self):
         with self.assertRaises(AttributeError):
             yield getstream.start(stream_info, name, "deadbeef" * 12, 0)
 
+    @defer.inlineCallbacks
+    def test_getstream_convert_fee_method(self):
+        getstream = self.init_getstream_with_mocs()
+        name = 'test'
+        stream_info = None
+
+        yield self.assertTrue(moc_convert_fee("USD", 60))
+        yield self.assertRaises(TypeError, getstream.convert_fee, "USD:60")
+        yield self.assertRaises(TypeError, getstream.convert_fee,
+                                "USD:60", "USD", 60)
```

Comment:
`assertTrue` and `assertRaises` do not return `Deferreds` so I'm not sure if these test even runs, did you try running the test? Put an `self.assertTrue(False)` at the end of the test, if the test doesn't fail then it's not getting run :-p

Reply:
Yep, I ran the test. It did indeed run locally for me and it seems Travis ran it too. 

Maybe I am not fully understanding what you are saying.